packages/cfd/src/types/cfd-store.types.ts

```diff
@@ -0,0 +1,86 @@
+import { DetailsOfEachMT5Loginid, Mt5NewAccount, VerifyEmailResponse } from '@deriv/api-types';
+
+type TStoreProofOfAddressArgs = {
+    file_uploader_ref: HTMLDivElement | null;
+    values: {
+        [key: string]: string;
+    };
+};
+
+export type TCFDStore = {
+    setMT5TradeAccount: (arg: any) => void;
+    toggleCFDVerificationModal: () => void;
+    setJurisdictionSelectedShortcode: (shortcode: string) => void;
+    setAccountType: (account_type: { category: string; type?: string }) => void;
+    dxtrade_tokens: {
+        demo: string;
+        real: string;
+    };
+    mt5_trade_account: Required<DetailsOfEachMT5Loginid>;
+    real_synthetic_accounts_existing_data: DetailsOfEachMT5Loginid & DetailsOfEachMT5Loginid[];
+    real_financial_accounts_existing_data: DetailsOfEachMT5Loginid & DetailsOfEachMT5Loginid[];
+    account_type: {
+        type: string;
+        category: string;
+    };
+    jurisdiction_selected_shortcode: string;
+    toggleJurisdictionModal: () => void;
+    has_submitted_cfd_personal_details: boolean;
+    is_jurisdiction_modal_visible: boolean;
+    clearCFDError: () => void;
+    current_list: Record<string, DetailsOfEachMT5Loginid & { enabled: number } & DetailsOfEachMT5Loginid[]>;
+    is_compare_accounts_visible: boolean;
+    toggleCompareAccounts: () => void;
+    dxtrade_companies: Record<string, any>;
```

Comment:
Please checkout this file for the type of this :pray: 
`packages/cfd/src/Stores/Modules/CFD/Helpers/cfd-config.ts`

Reply:
Thank you 🙏